src/microbe_stage/PatchMap.cs

```diff
@@ -7,8 +7,18 @@
 /// <summary>
 ///   A container for patches that are joined together
 /// </summary>
-public class PatchMap
+[UseThriveSerializer]
+public class PatchMap : ISaveLoadable
 {
+    [JsonProperty]
+    public Random Seed = null!;
+
+    [JsonProperty]
+    public bool NeedsVent = true;
+
+    [JsonProperty]
+    public bool NeedsTidepool = true;
```

Comment:
Also should be properties probably. Or what these are needed for? I feel like these are much more about the map generation and **should not** be stored in the actual finished map.

Reply:
New game settings. depending on new game settings you might need that or not.

Reply:
New game settings should be passed to the map generator, which then can look at this info. No need to save this in the *created* map.